strongbox-data-service/src/main/java/org/carlspring/strongbox/config/EmbeddedOrientDbConfig.java

```diff
@@ -40,9 +41,12 @@ OrientDB orientDB()
 
         if (!orientDB.exists(database))
         {
-            logger.info(String.format("Creating database [%s]...", database));
+            logger.info(String.format("Unpacking database from snapshot [%s]...", database));
 
-            orientDB.create(database, ODatabaseType.PLOCAL);
+            try (InputStream is = new ClassPathResource("db/snapshot/strongbox-db-snapshot.zip").getInputStream())
```

Comment:
why can't we use `java.util.zip.ZipInputStream` here instead of `ZipUtil`?

Reply:
I mean that we can avoid additional `<artifactId>zt-zip</artifactId>` dependency in this case.

Reply:
Because, as you can see, usage of `<artifactId>zt-zip</artifactId>` is simple. No exceptions to catch/throw, no streams to handle, just one line of code. 

You can read more about it [here](https://github.com/zeroturnaround/zt-zip#user-content-background).